modules/fts/pages/fts-accessing-managing-full-text-indexes.adoc

```diff
@@ -0,0 +1,20 @@
+= Accessing and Managing Full Text Indexes
+
+[abstract]
+Full Text Indexes are different from the _Global_ indexes that are accessed under the *Indexes* tab in the left-hand navigation panel of the Couchbase Web Console.
```

Comment:
If you're going to specify here that full text indexes are "different" from the global (gotta mention _secondary_ also here) indexes - then this might be a good place to explain how they're different as well.

Don't think we've documentation anywhere on why and how the FTS indexes differ from global secondary indexes.